pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -415,7 +415,7 @@ type KubeletConfiguration struct {
 	// enable gathering custom metrics.
 	EnableCustomMetrics bool `json:"enableCustomMetrics"`
 	// Comma-delimited list of hard eviction expressions.  For example, 'memory.available<300Mi'.
-	EvictionHard string `json:"evictionHard"`
+	EvictionHard *string `json:"evictionHard"`
```

Comment:
@derekwaynecarr recently enabled eviction by default by setting the  `EvictionHard` threshold, so now `""` is no longer the correct default. But `""` is still a valid value, because, presumably, setting `""` is how you would disable eviction. So we need a pointer on the external. I can definitely add `omitempty` though.